client/src/pages/Meals.js

```diff
@@ -1,35 +1,94 @@
+        return abort;
+    }, [cuisines]);
+
     return (
-        <React.Fragment>
+        <Main>
             <ResponsiveSideBar>
-                <Typography>Responsive Side Bar</Typography>
+                <Grid container spacing={2} alignContent="flex-start">
+                    <Grid item xs={12}>
+                        <Typography>Cuisine:</Typography>
+                        {[...cuisines].map((cuisine) => (
+                            <Button
+                                key={cuisine}
+                                color="primary"
+                                variant="contained"
+                                onClick={() => removeCuisine(cuisine)}
+                            >
+                                <Typography variant="button">{cuisine}</Typography>
+                            </Button>
+                        ))}
+                    </Grid>
+                    <Grid item xs={12}>
+                        {[...availableCuisines].map((cuisine) => (
+                            <Button
+                                key={cuisine}
+                                variant="contained"
+                                onClick={() => addCuisine(cuisine)}
+                            >
+                                <Typography variant="button">{cuisine}</Typography>
+                            </Button>
+                        ))}
+                    </Grid>
+                </Grid>
             </ResponsiveSideBar>
-            <Main>
-                {meals.map((meal) => {
-                    return (
-                        <MealCard
-                            key={meal.id}
-                            id={meal.id}
-                            mealPic={meal.mealPic}
-                            title={meal.title}
-                            price={meal.price}
-                            chefName={meal.chefName}
-                            chefId={meal.chefId}
-                            chefPic={meal.chefPic}
-                            location={meal.location}
-                        />
-                    );
-                })}
-            </Main>
-        </React.Fragment>
+            <Grid container spacing={4} alignContent="flex-start">
+                <Grid item xs={12}>
+                    <Typography className={classes.availableMeals}>
+                        Available Meals:
+                    </Typography>
+                </Grid>
+                {meals.map((meal) => (
+                    <MealCard
+                        key={meal._id}
+                        id={meal._id}
+                        mealPic={meal.mealPic}
+                        title={meal.title}
+                        price={meal.price}
+                        chefName={meal.chefName}
+                        chefId={meal.chefId}
+                        chefPic={meal.chefPic}
+                        location={meal.location}
+                    />
```

Comment:
Is there a reason to pass each of these individuall vs just passing meal itself?

Reply:
fixed it!